### ISSUE TITLE
Menu not working on mobile browsers

### ISSUE THREAD

**Issue by scurfin** [notandigg/grimsi-geysir#29]
Wow, thanks for your quick fixes / changes.

I've tested your latest version and noticed that the menu doesn't work on mobile browsers like Chrome or Firefox. Pressing the button only reloads the site.
With the browser option "show desktop version" the menu works as expected.

**Comment by notandigg** [comment]
I just got Google Chrome on my Iphone and tested it out, I can't replicate what you are experiencing. On what mobile phone have you seen this behavior? And this can't have happened after this latest release since I haven't changed anything regarding the menu for quite some time.

**Comment by scurfin** [comment]
I found the cause on my page. I use a logo with a transparent background that overlaps the menu button. Instead of pressing the menu button, I pressed the logo and reloaded the page.

The issue can be closed.

Perhaps as an improvement for future versions it would be possible to add an option or CSS setting for the dynamic resizing of the logo based on the screen size?
Currently, the logo on desktop pages is relatively small, while in mobile browsers it takes up the entire width.
